test/test_integration.rb

```diff
@@ -198,12 +198,13 @@ def test_phased_restart_via_pumactl
         done = true
       end
     end
+    assert File.exist? @bind_path
```

Comment:
Why did moving this before the CLI invocation matter? I assume the state would be the same.

Reply:
I ran thru several iterations in my fork.  At one point, the file did not exist after stop.  An, since the issue was that sockets were disappearing after a restart, I moved it.

Reply:
I moved this back, I must have moved before I got Binder working correctly.